src/bundle/Resources/views/content/locationview.html.twig

```diff
@@ -110,6 +110,7 @@
 
 {% block javascripts %}
     {%  javascripts
+        '@EzPlatformAdminUiBundle/Resources/public/js/scripts/services/notification.service.js'
```

Comment:
I think this should be in `layout.html.twig` to make this global not only for `content structure`.